src/Service.php

```diff
@@ -17,7 +17,7 @@
     ],
     steps: [
         "extractor" => "extractor",
-        "transformer" => "lookup",
+        "lookup" => "transformer",
         "loader" => "loader",
```

Comment:
Il serait temps de changer ces annotations et supprimer la couche de compatibilité PHP 8.0.

```suggestion
        new Configurator\Pipeline\StepExtractor('extractor'),
        new Configurator\Pipeline\StepTransformer('lookup'),
        new Configurator\Pipeline\StepLoader('loader'),
```